web-client/iron-remote-desktop/src/interfaces/DesktopSize.ts

```diff
@@ -1,4 +1,6 @@
 ﻿export interface DesktopSize {
     width: number;
     height: number;
+
+    construct(width: number, height: number): DesktopSize;
```

Comment:
note: It seems the position of this item is inconsistent across the interfaces.

Reply:
Fixed